docs/setup/settings.asciidoc

```diff
@@ -82,52 +83,59 @@ Elasticsearch nodes on startup.
 `elasticsearch.sniffOnConnectionFault:`:: *Default: false* Update the list of
 Elasticsearch nodes immediately following a connection fault.
 
-`elasticsearch.ssl.alwaysPresentCertificate:`:: *Default: false* Controls whether to always present the certificate specified by
-`elasticsearch.ssl.certificate` or `elasticsearch.ssl.keystore.path` when requested. This setting applies to all requests to Elasticsearch,
-including requests that are proxied for end users. Setting this to `true` when Elasticsearch is using certificates to authenticate users can
-lead to proxied requests for end users being executed as the identity tied to the configured certificate.
+`elasticsearch.ssl.alwaysPresentCertificate:`:: *Default: false* Controls {kib}'s behavior in regard to presenting a client certificate when
+requested by {es}. This setting applies to all outbound SSL/TLS connections to {es}, including requests that are proxied for end users.
++
+WARNING: Setting this to `true` when {es} is using certificates to authenticate end users with a PKI realm can lead to proxied requests for
+end users being executed as the identity tied to the {kib} server.
 
-`elasticsearch.ssl.certificate:` and `elasticsearch.ssl.key:`:: Paths to a PEM-encoded X.509 certificate and its private key, respectively.
-When `xpack.security.http.ssl.client_authentication` in Elasticsearch is set to `required` or `optional`, the certificate and key are used
-to prove Kibana's identity when it makes an outbound request to your Elasticsearch cluster.
+`elasticsearch.ssl.certificate:` and `elasticsearch.ssl.key:`:: Paths to a PEM-encoded X.509 client certificate and its corresponding
+private key. These are used by {kib} to authenticate itself when making outbound SSL/TLS connections to {es}. For this setting to take
+effect, the `xpack.security.http.ssl.client_authentication` setting in {es} must be also be set to `"required"` or `"optional"` to request a
+client certificate from {kib}.
 +
---
 NOTE: These settings cannot be used in conjunction with `elasticsearch.ssl.keystore.path`.
---
 
-`elasticsearch.ssl.certificateAuthorities:`:: Paths to one or more PEM-encoded X.509 certificates. These certificates may consist of a root
-certificate authority (CA), and one or more intermediate CAs, which make up a trusted certificate chain for Kibana. This chain is used to
-establish trust when Kibana creates an SSL connection with your Elasticsearch cluster. In addition to this setting, trusted certificates may
-be specified via `elasticsearch.ssl.keystore.path` and/or `elasticsearch.ssl.truststore.path`.
+`elasticsearch.ssl.certificateAuthorities:`:: Paths to one or more PEM-encoded X.509 certificate authority (CA) certificates which make up a
+trusted certificate chain for {es}. This chain is used by {kib} to establish trust when making outbound SSL/TLS connections to {es}.
++
+In addition to this setting, trusted certificates may be specified via `elasticsearch.ssl.keystore.path` and/or
```

Comment:
> In addition to this setting, ....

This seems to imply that you can use all three settings at the same time. Is that correct? If not, I'd recommend "Alternatively, you can specify trusted certificates in ..."

Reply:
Yes, you can use all three at the same time if you wanted. It's a unlikely scenario but it is possible that you may have some CA certificates in PEM format and some other ones in a PKCS#12 trust store. It was a lot simpler logic to combine than all than to try to throw an error if CA certs were detected in multiple places.